WordPress/src/androidTest/java/org/wordpress/android/e2e/ContactUsTests.java

```diff
@@ -0,0 +1,84 @@
+package org.wordpress.android.e2e;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.wordpress.android.R;
+import org.wordpress.android.e2e.flows.LoginFlow;
+import org.wordpress.android.e2e.pages.ContactSupportScreen;
+import org.wordpress.android.support.BaseTest;
+
+import static org.wordpress.android.BuildConfig.E2E_WP_COM_USER_EMAIL;
+import static org.wordpress.android.support.WPSupportUtils.pressBackUntilElementIsDisplayed;
+
+public class ContactUsTests extends BaseTest {
+    @Before
+    public void setUp() {
+        logoutIfNecessary();
+    }
+
+    @After
+    public void tearDown() {
+        pressBackUntilElementIsDisplayed(R.id.continue_with_wpcom_button);
+    }
+
+    @Test
+    public void sendButtonEnabledWhenTextIsEntered() {
+        try {
+            new LoginFlow()
+                .chooseContinueWithWpCom()
+                .tapHelp()
+                .assertHelpAndSupportScreenLoaded()
+                .openContactUs()
+                .assertContactSupportScreenLoaded()
+                .assertSendButtonDisabled()
+                .setMessageText("Hello")
+                .assertSendButtonEnabled()
+                .setMessageText("")
+                .assertSendButtonDisabled();
+        } finally {
+            new ContactSupportScreen().goBackAndDeleteUnsentMessageIfNeeded();
+        }
+    }
+
+    @Ignore("As long as CI does not use gradle.properties from MobileSecrets")
```

Comment:
As can be seen from the number of commits in this PR starting with `CI fail fix:...`, I spent quite a while figuring out why the tests that involve `Contact Support` screen failed on CI but never failed the same way locally. At some point (thanks to video recording in Firebase) I saw that the screen on CI:

<img width="319" alt="Screenshot 2021-12-23 at 19 58 29" src="https://user-images.githubusercontent.com/73365754/147283943-4681c1a2-d910-4602-91e9-f87f37198b49.png">

Looks different from the same screen on my local Emulator:

<img width="319" alt="Screenshot 2021-12-23 at 19 52 52" src="https://user-images.githubusercontent.com/73365754/147283988-ec635f5b-355c-4a5c-9d08-3e1f7a2392d4.png">

After blaming the Emulator/Android differences for some time, I realized that CI uses the default `gradle.properties`, which makes the screen look like differently (and the message cannot be sent, hence the disabled test).

Since I automated this test, I decided to keep it for the case if Mobile Secrets will be used on CI.

Reply:
In case we have this test running in CI are we going to be creating real Zendesk tickets? I'm afraid it can get real noisy.

Reply:
I have this suspicion too. This test is disabled for now, so it's not an urgent matter, but I think we can contact the HE team, and ask them if there's a painless way for them to filter out certain messages, so that they wouldn't be bothered with them.

Reply:
Yeah, I agree.

Reply:
If it turns out to be the case that we cannot prevent the creation of real tickets, and idea could be to use the Zendesk APIs to [find the tickets](https://developer.zendesk.com/api-reference/ticketing/tickets/tickets/#list-tickets) created by the UI tests and [delete](https://developer.zendesk.com/api-reference/ticketing/tickets/tickets/#delete-ticket) them.